news/2017-12-24-winter-themed-spotlights-2017.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Winter Themed Spotlights 2017"
+date: 2017-12-24 13:31:35 +0000
+---
+
+Greetings everyone! We hope all of you have a wonderful and snowy Christmas today and celebrate it with loved ones. For this special day, the Spotlights Team prepared a winter themed spotlight for those who feel like only presents are just not enough. Let's drink some hot chocolate at this special day while clicking those circles; hitting the drum; catching those healthy fruits and go octopus-mode with the squares!
```

Comment:
was there a reason to split the list with semicolons (you'd only use semicolons in lists for when the list is too complicated with commas)